react/tic-tac-toe/src/app/screens/Game/index.js

```diff
@@ -56,21 +59,20 @@ class Game extends Component {
     const status = winner ? `Winner: ${winner}` : `${player}'s turn`;
 
     return (
-      <div className={styles.game}>
-        <div className={styles.gameBoard}>
-          <Board
-            squares={current.squares}
-            onClick={this.handleClick}
-          />
-        </div>
-        <div className={styles.gameInfo}>
-          <div>{status}</div>
-          <HistoryList
-            history={history}
-            onChangeHistory={this.handleJumpTo}
-          />
+      <Provider store={store}>
+        <div className={styles.game}>
+          <div className={styles.gameBoard}>
+            <Board squares={current.squares} onClick={this.handleClick} />
+          </div>
+          <div className={styles.gameInfo}>
+            <div>{status}</div>
```

Comment:
this should not be a div, only use div for generic containers. This is a span, or maybe an h2 or h3